### ISSUE TITLE
[1.16.2] v0.8 - Linking an energy import pedestal to a energy smelter caused Ticking Block crash loop

### ISSUE THREAD

**Issue by scopiplays** [Mowmaster/Pedestals#12]
First time using your mod.  This is from the All the Mods 1.6.2 release v1.0.3

I put a pedestal next to a RF Tools powercube and added the energy import upgrade.
I linked that pedestal to a pedestal with the energy smelter upgrade, and the server went boom.

Here is the crashlog:

[crash-2020-09-06_20.54.32-server.txt](https://github.com/Mowmaster/Pedestals/files/5180291/crash-2020-09-06_20.54.32-server.txt)

**Comment by Mowmaster** [comment]
the pedestal with the energy import was placed against the power cube or the power cube was inserted into the pedestal?

**Comment by scopiplays** [comment]
hmm. i tried to recreate it.  the pedestal did not take energy from the power cube when placed next to it.  inserting the cube causes crash.

**Comment by Mowmaster** [comment]
ah, this is an ATM pack thing, testing with the mod alone and with other packs this issue doesnt happen. chances are theres a mod in the pack causing this to happen.
Regardless, I am going to try something to see if it can be avoided on my end as well.

**Comment by Mowmaster** [comment]
Welp doing some testing and this is only a 1.16.2 issue, still looking into whats wrong...

**Comment by Cntrldfusion** [comment]
I had the same thing occur.  I had a pedestal with an energy import placed on the right side of an RFTools Coal Generator and it was not pulling out any energy.  So I created a RFTools powercube, set the right side to output, placed the pedestal with the energy import onto that side of the power cube and when I went to link it to a pedestal with the energy relay upgrade the server went into a crash loop.   There's no picture of the setup since I had to restore form a 2 hr old backup but I will attach the crash report in case that helps.

[crash-2020-09-28_17.21.12-server.txt](https://github.com/Mowmaster/Pedestals/files/5295564/crash-2020-09-28_17.21.12-server.txt)
